src/main/java/mSearch/filmlisten/WriteFilmlistJson.java

```diff
@@ -22,112 +22,163 @@
-                        if (datenFilm.arr[m].equals(thema)) {
-                            jg.writeString("");
+                    jg.writeString(DatenFilm.COLUMN_NAMES[DatenFilm.JSON_NAMES[i]]);
+                }
+                jg.writeEndArray();
+                //Filme schreiben
+                for (DatenFilm datenFilm : listeFilme) {
+                    datenFilm.arr[DatenFilm.FILM_NEU] = Boolean.toString(datenFilm.isNew()); // damit wirs beim nächsten Programmstart noch wissen
+
+                    jg.writeArrayFieldStart(DatenFilm.TAG_JSON_LIST);
+                    for (int i = 0; i < DatenFilm.JSON_NAMES.length; ++i) {
+                        int m = DatenFilm.JSON_NAMES[i];
+                        if (m == DatenFilm.FILM_SENDER) {
+                            if (datenFilm.arr[m].equals(sender)) {
+                                jg.writeString("");
+                            } else {
+                                sender = datenFilm.arr[m];
+                                jg.writeString(datenFilm.arr[m]);
+                            }
+                        } else if (m == DatenFilm.FILM_THEMA) {
+                            if (datenFilm.arr[m].equals(thema)) {
+                                jg.writeString("");
+                            } else {
+                                thema = datenFilm.arr[m];
+                                jg.writeString(datenFilm.arr[m]);
+                            }
                         } else {
-                            thema = datenFilm.arr[m];
                             jg.writeString(datenFilm.arr[m]);
                         }
-                    } else {
-                        jg.writeString(datenFilm.arr[m]);
                     }
+                    jg.writeEndArray();
                 }
-                jg.writeEndArray();
+                jg.writeEndObject();
+                Log.sysLog("   --> geschrieben!");
             }
-            jg.writeEndObject();
-            Log.sysLog("   --> geschrieben!");
         } catch (Exception ex) {
             Log.errorLog(846930145, ex, "nach: " + datei);
-        } finally {
-            try {
-                if (jg != null) {
-                    jg.close();
-                }
-                // die werden nicht immer korrekt geschlossen !??!
-                if (zipOutputStream != null) {
-                    zipOutputStream.close();
-                }
-                if (xZOutputStream != null) {
-                    xZOutputStream.close();
-                }
-            } catch (Exception e) {
-                Log.errorLog(732101201, e, "close stream: " + datei);
+        }
+    }
+
+    private Path testNativeXz() {
+        final String[] paths = {"/usr/bin/xz", "/opt/local/bin/xz", "/usr/local/bin/xz"};
+
+        Path xz = null;
+
+        for (String path : paths) {
+            //System.out.println("Path: " + path);
+            xz = Paths.get(path);
+            if (Files.isExecutable(xz)) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)